.github/workflows/common.yml

```diff
@@ -7,23 +7,28 @@ on:
       - release-*
     tags:
       - v[0-9]+.[0-9]+.[0-9]+*
+
   pull_request:
 
+  workflow_call:
+    inputs:
+      go-version:
+        description: An array of Go versions to be tested against, in a format of json string.
+        required: false
+        type: string
+        default: '["tip"]'
+
 jobs:
   check:
     name: Checks
     runs-on: ubuntu-22.04
     strategy:
+      fail-fast: false
       matrix:
-        go-version: [1.18.x]
-        may-fail: [false]
```

Comment:
I think we should change the behavior for standard jobs. it was failing on error, but now it doesn't